### ISSUE TITLE
Build failure against boost-1.70 in server/supernova/sc/sc_osc_handler.cpp:891:83: error: 'boost::asio::ip::tcp::acceptor' has no member named 'get_io_service'

### ISSUE THREAD

**Issue by a17r** [supercollider/supercollider#4490]
## Environment

* SuperCollider version: 3.10.2, patched for boost-1.67 (de567f91)
* Operating system: Gentoo Linux
* Other details (Qt version, audio driver, etc.): Qt 5.12.4

Downstream report including full build.log: https://bugs.gentoo.org/689244
```
SuperCollider-Source/server/supernova/sc/sc_osc_handler.cpp: In member function 'void nova::detail::sc_osc_handler::start_tcp_accept()':
SuperCollider-Source/server/supernova/sc/sc_osc_handler.cpp:891:83: error: 'boost::asio::ip::tcp::acceptor' {aka 'class boost::asio::basic_socket_acceptor<boost::asio::ip::tcp>'} has no member named 'get_io_service'
     tcp_connection::pointer new_connection = tcp_connection::create(tcp_acceptor_.get_io_service());
                                                                                   ^~~~~~~~~~~~~~
```

boost documentation seems to suggest to use `get_executor().context()` instead.

**Comment by brianlheim** [comment]
thanks! please submit a patch if you have time.

**Comment by brianlheim** [comment]
according to this stack overflow answer you are right as well: https://stackoverflow.com/questions/39445636/alternative-to-deprecated-get-io-service

**Comment by a17r** [comment]
Indeed that's what I've been preparing, though I am working against 3.10 branch. Should I rebase on `develop`?

**Comment by brianlheim** [comment]
3.10 is good!

**Comment by brianlheim** [comment]
Closed in #4612
